portfolio/src/main/webapp/script.js

```diff
@@ -12,24 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* eslint-disable no-unused-vars */
+
 let slideIndex = 0;
 
 /**
  * Helper function for naviagtion via arrows.
+ * @param {number} offset - the number of slides to change by
+ * (negative numbers indicate going backwards).
  */
 function changeSlides(offset) {
   showSlides(slideIndex + offset);
 }
 
 /**
- * Displays photo at new index by changing its display style and setting the corresponding indicator.
+ * Displays photo at new index by changing its display style
+ * and setting the corresponding indicator.
+ * @param {number} newIndex - Index of photo to be displayed.
  */
 function showSlides(newIndex) {
-  let slides = document.getElementsByClassName("slide");
-  let dots = document.getElementsByClassName("dot");
+  const slides = document.getElementsByClassName('slide');
```

Comment:
Constant's name should be written in upper case with _.

Reply:
Do they have to be uppercase every time `const` is used? Because I wouldn't say these are constants in the classical sense, which is why I used `let` previously but then the linter complains since they aren't modified.